src/current/cockroachcloud/cluster-management.md

```diff
@@ -68,53 +68,50 @@ To add or remove nodes from your cluster:
 
     The **Edit cluster** page displays.
```

Comment:
This section sounds like it's specifically for "Add or remove nodes from a cluster". Just double checking.